src/main/java/de/tum/in/www1/artemis/web/rest/StudentExamResource.java

```diff
@@ -717,11 +717,12 @@ public ResponseEntity<StudentExam> submitStudentExam(@PathVariable Long courseId
         if (studentExam.isSubmitted()) {
             throw new BadRequestException();
```

Comment:
It would probably make sense to also change this into a `BadRequestAlertException` and provide more feedback.